tt_metal/include/compute_kernel_api/tilize.h

```diff
@@ -119,6 +138,11 @@ ALWI void unpack_tilizeA_B_block(uint32_t icb0, uint32_t icb1, uint32_t block, u
     UNPACK(( llk_unpack_tilizeA_B_block(icb0, icb1, block, tile_idx_b, num_faces) ));
 }
 
```

Comment:
Add some comments to explain how this new api works

Reply:
I've commented the APIs I've added, have a look now